app/internal/share_event.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Dict, Union
```

Comment:
I believe this file should be in `app/routers/share.py`?

https://fastapi.tiangolo.com/tutorial/bigger-applications/